.github/workflows/app-build-verify.yml

```diff
@@ -110,24 +117,24 @@ jobs:
           **/pyproject.toml
           .pre-commit-config.yaml
 
-    - name: Get Packages
+    - name: Get Briefcase Package
       # Briefcase will build and package itself in a previous step in its CI
       if: endsWith(inputs.repository, 'briefcase')
       uses: actions/download-artifact@v3.0.2
       with:
-        name: packages-${{ steps.package.outputs.name }}
+        name: packages-briefcase
         path: dist
 
     - name: Install Briefcase Artefact
       if: endsWith(inputs.repository, 'briefcase')
-      run: python -m pip install dist/briefcase-*.whl
+      run: python3 -m pip install dist/briefcase-*.whl
```

Comment:
Realized I've been making a flawed assumptions with the gating of some of these steps.

These checks on the name of the repository just assume that forks will use the same name as the repos in the beeware org. But there's definitely zero requirement on that.

Checks like this should be updated to checking the name of the target repo.

I think the closest thing to that is the "base" of a PR accessible here:
```
${{ github.event.pull_request.base.repo.full_name }}
```
This would yield, for instance, `beeware/briefcase` for PRs for Briefcase irrespective of the name of the forked repo.

Reply:
However, the CI workflow obviously also runs for events that aren't for PRs. So, an ultimate solution will need to fallback to other values in that situation.

(mostly notes to myself....debating about addressing this now or not....this hasn't been an actual issue yet....afaik)

Reply:
Agreed this is technically an issue, and I'm totally open to fixes - but I'm also not exceptionally concerned by it. The set of people who change the name of a repo on fork is... small. I and they can contribute the fix if they're bitten by this problem.